File: src/main/java/no/nav/syfo/domain/dialogmelding/RSMottaker.java
package no.nav.syfo.domain.dialogmelding;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class RSMottaker {
    private String partnerId;
    private String herId;
    private String orgnummer;
    private String navn;
    private String adresse;
    private String postnummer;
    private String poststed;
    private RSBehandler behandler;
}

File: src/main/kotlin/no/nav/syfo/dialogmelding/domain/RSMottaker.kt
package no.nav.syfo.dialogmelding.domain

data class RSMottaker(
    val partnerId: String,
    val herId: String,
    val orgnummer: String? = null,
    val navn: String,
    val adresse: String? = null,
    val postnummer: String? = null,
    val poststed: String? = null,
    val behandler: RSBehandler,
)
